static/code/stackblitz/v7/react/index.html

```diff
@@ -1,8 +1,26 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  </head>
+    <meta charset="utf-8" />
+    <title>Ionic App</title>
+
+    <base href="/" />
 
-  <div id="root"></div>
+    <meta name="color-scheme" content="light dark" />
+    <meta
+      name="viewport"
+      content="viewport-fit=cover, width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <meta name="format-detection" content="telephone=no" />
+    <meta name="msapplication-tap-highlight" content="no" />
+
+    <!-- add to homescreen for ios -->
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-title" content="Ionic App" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black" />
+  </head>
```

Comment:
Why did we decide to add all of this for React when we don't do it for the other ones?

Examples:

**Vue**
- [Vue Stackblitz](https://github.com/ionic-team/ionic-docs/blob/main/static/code/stackblitz/v7/vue/index.html)
- [Vue Starter](https://github.com/ionic-team/starters/blob/main/vue-vite/base/index.html)

**Angular**
- [Angular Stackblitz](https://github.com/ionic-team/ionic-docs/blob/aef7605237b5294df12989ba7ecf0f1c6bdad9aa/src/components/global/Playground/stackblitz.utils.ts#L135)
- [Angular Starter](https://github.com/ionic-team/starters/blob/main/angular/base/src/index.html)

Reply:
I noted this in the PR description:

> Updated index.html to more closely align with the react-vite starters

My goal with this is to make it easier to update these playgrounds moving forward. So when we update the starters for v8 it should be a fairly easy copy and paste for the playgrounds too.